helios-client/src/main/java/com/spotify/helios/client/HeliosClient.java

```diff
@@ -441,15 +441,25 @@ public Integer apply(final Response reply) {
   }
 
   public ListenableFuture<HostStatus> hostStatus(final String host) {
-    return get(uri(path("/hosts/%s/status", host)), HostStatus.class);
+    return hostStatus(host, Collections.<String, String>emptyMap());
+  }
+
+  public ListenableFuture<HostStatus>
+  hostStatus(final String host, final Map<String, String> queryParams) {
```

Comment:
I made it a map in case we want to add more query parameters in the future. And `uri()` takes a map anyways.